package-lock.json

```diff
@@ -1,40 +1,104 @@
 {
     "name": "@oat-sa/tao-core-sdk",
-    "version": "3.0.0",
-    "lockfileVersion": 1,
+    "version": "3.1.0",
+    "lockfileVersion": 3,
```

Comment:
You updated the lock file version. Is it safe?

Reply:
Reverted in [8dd164a](https://github.com/oat-sa/tao-core-sdk-fe/pull/181/commits/8dd164a5ab2a1291f2955696cf276d1a29a48f5a)